package.json

```diff
@@ -17,17 +18,16 @@
   },
   "homepage": "https://github.com/EventMobi/eslint-config-eventmobi#readme",
   "dependencies": {
-    "eslint-config-airbnb": "11.1.0",
-    "eslint-config-airbnb-base": "7.1.0"
+    "eslint-config-airbnb": "14.0.0",
+    "eslint-config-airbnb-base": "11.0.1"
   },
   "peerDependencies": {
-    "eslint": "^3.5.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
-    "eslint-plugin-import": "^1.6.0",
-    "eslint-plugin-react": "^6.0.0"
+    "eslint": "^3.13.0"
   },
   "devDependencies": {
-    "eslint": "^3.5.0",
-    "eslint-plugin-import": "^1.13.0"
+    "eslint": "^3.13.0",
```

Comment:
Need to add `"protractor": true` to your .eslintrc in `globals`. There is also this: https://github.com/alecxe/eslint-plugin-protractor which may work on your current version - it may add the globals too and some other nice things. You'd just add it to plugins in your .eslintrc.

It looks like you guys are on `0.2.0` (which is from  Oct 29, 2015) so updating likely requires some work - I assume that's why you're behind.